### PR TITLE
Feat initial detailpage

### PR DESCRIPTION
**Pull request by kieranslost**
Das isch mis erste mal wo ich das mach. Mach ich das richtig so? 

Anyway das hani kmacht:

Started working on the detail-page. Almost finished btw, ik i'm increadible. I know how to style a page that doesn't do shit 😎

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .idea
-**/.DS_Store
\ No newline at end of file
+**/.DS_Store
+node_modules
```
Comment (deepspace1000) [inline .gitignore:3]: not needed we dont have node modules in the root directory. you probably accidentally executed yarn in the root. you can delete the packes.json files in the root directory

File: frontend/src/pages/DetailPage.tsx [hunk 2]
```diff
@@ -1,7 +1,65 @@
...
+
+const BottomImage = styled('img')`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  height: 30rem;
+`;
+
+/*
+const Button = styled('button')`
```
Comment (deepspace1000) [inline frontend/src/pages/DetailPage.tsx:40]: delete unused code

File: frontend/src/pages/HomePage.tsx [hunk 3]
```diff
@@ -34,15 +34,68 @@ export function HomePage() {
...
+            }}>
+            Return Home
+          </Button>
+        </a>
+      </Grid>
     </>
   );
 }
 
+/*<ButtonImage src="../assets/homepage/ReturnHome.png"/>*/
```
Comment (deepspace1000) [inline frontend/src/pages/HomePage.tsx:93]: delete unused code

File: frontend/src/pages/HomePage.tsx [hunk 4]
```diff
@@ -56,19 +109,31 @@ const FrontImage2 = styled(FrontImage)`
...
   left: 15em;
 `;
-
 const FrontImage3 = styled(FrontImage)`
   top: -3em;
   left: 25em;
 `;
 
-const TextContainer = styled('div')`
+const ServiceIcons = styled('img')``;
```
Comment (deepspace1000) [inline frontend/src/pages/HomePage.tsx:117]: i guess you don't need to use styled elements when you don't give it any styles

File: package-lock.json
Comment (deepspace1000) [inline package-lock.json:1]: Delete. should not exist

File: package.json
Comment (deepspace1000) [inline package.json:1]: Delete, should not exist

**Review by deepspace1000 — COMMENTED** [review]
I didn't really look at the code just some points that stood out.
Please sqash all your 7 commits in to one and give it a nice commit message 😉

File: .gitignore (hunk 1 above)
Comment (kieranslost) [reply .gitignore:3]: genau das isch passiert, wenn ich mich richtig erinnere sogar de martin :).

Comment (deepspace1000) [reply .gitignore:3]: Ja im au er het node modules im root pushed gha 😅. Ich hans aber grad vorher schnell glöscht. jetzt muesch es nur no bi dir au lösche.

File: .gitignore
Comment (deepspace1000) [inline .gitignore:1]: Well we still need the gitignore for the idea and DS_Store directory 😂 just not the node_modules

File: frontend/.gitignore [hunk 5]
```diff
@@ -17,7 +17,7 @@ dist-ssr
 # Editor directories and files
 .vscode/*
 !.vscode/extensions.json
-.idea
+../.idea/
```
Comment (deepspace1000) [inline frontend/.gitignore:20]: creative but hard to read or for someone to find out why the folder is ignored. its best practice to have the gitignore in the directory in which you want to ignore files

**Review by deepspace1000 — COMMENTED** [review]
Now you have all of the images twice. the `public` folder is for different stuff, like tab icons. you can delete the files in the `public` folder and just use the ones in the `assets` folder

**Review by deepspace1000 — APPROVED** [review]
Fine for me. just delete the package.json and package-lock.json in the root directory.

**Pull request merged**